libqtile/widget/wttr.py

```diff
@@ -0,0 +1,87 @@
+from urllib.parse import quote, urlencode
+
+from libqtile.widget import GenPollUrl, base
+
+
+class Wttr(GenPollUrl):
```

Comment:
`GenPollUrl` inherits `ThreadPoolText` which has a default `update_interval` of 600 seconds: https://github.com/qtile/qtile/blob/7ec0bdd002ea44417bb2c439aca466f382ce80a5/libqtile/widget/base.py#L515-L518

Is this something that you want your users to be able to update? If so, add it to your `defaults` so users know they can change it (it'll still work if they set it in their config but this would make it clearer in the docs).

Reply:
I forgot about it. I add it now.
And something else. Are docstrings without mistakes? My English is not very good.

Reply:
Sorry - my advice has been terrible! I've just generated the docs for your PR and can see that it adds all the defaults from inherited classes, so `json` is listed etc.

That's a little bit frustrating because you won't want users to change most of those.

Let me ask a question to the devs on this.

Reply:
Will waiting an answer.)

Reply:
LGTM, I don't think the json-rendering-in-docs issue is a problem you need to solve for this widget. Thanks!

Reply:
Sounds good. 

Also, would just add that I like this widget. Very neat. Thanks @akkaky

Reply:
Wow! Thanks! My first approved PR!